guide/ops/troubleshooting/detailed-support-report.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: website-normal
+title: Detailed Support Report
+toc: /guide/toc.json
+---
+
+If you wish to send a detailed report, then depending on the nature of the problem, consider 
+collecting the following information.
+
+See [Brooklyn Slow or Unresponse](slow-unresponsive.html) docs for details of these commands.
+ 
+{% highlight bash %}
+BROOKLYN_HOME=/home/users/brooklyn/apache-brooklyn-0.9.0-bin
+BROOKLYN_PID=$(cat $BROOKLYN_HOME/pid_java)
+REPORT_DIR=/tmp/brooklyn-report/
+
+uname -a > ${REPORT_DIR}/uname.txt
+df -h > ${REPORT_DIR}/df.txt
+cat /proc/cpuinfo > ${REPORT_DIR}/cpuinfo.txt
+cat /proc/meminfo > ${REPORT_DIR}/meminfo.txt
+ulimit -a > ${REPORT_DIR}/ulimit.txt
+top -n 1 -b > ${REPORT_DIR}/top.txt
+lsof -p ${BROOKLYN_PID} > ${REPORT_DIR}/lsof.txt
+netstat -an > ${REPORT_DIR}/netstat.txt
+
+jmap -histo:live $BROOKLYN_PID > ${REPORT_DIR}/jmap-histo.txt
+jmap -heap $BROOKLYN_PID > ${REPORT_DIR}/jmap-heap.txt
+for i in {1..10}; do
+  jstack ${BROOKLYN_PID} > ${REPORT_DIR}/jstack.${i}.txt
+  sleep 1
+done
+tar czf brooklyn-report.tgz ${REPORT_DIR}
+{% endhighlight %}
+
+Also consider providing your log files and persisted state, though extreme care should be taken if
```

Comment:
Alternatively could include "grep 'brooklyn gc' cloudsoft-amp.debug.log` which won't contain any sensitive information but will provide useful feedback on the memory pressure of the heap.